### PR TITLE
add rounding modes ROUND_UP and ROUND_DOWN

### PR DESCRIPTION
**Pull request by frederikbosch**
There are use cases for both rounding up and rounding down.

### CODE REVIEW

**Comment by frederikbosch** [comment]
@mathiasverraes If this PR is going to be merged, I would be happy to add another PR for the `nextrelease` branche.

File: lib/Money/Money.php [hunk 1]
```diff
@@ -261,6 +263,26 @@ public function allocate(array $ratios)
         return $results;
     }
 
+    /**
+     * @param int|float $amount
+     * @param $rounding_mode
+     * @return int
+     */
+    private function round ($amount, $rounding_mode)
```
Comment (camspiers) [inline lib/Money/Money.php]: Minor, remove the space `round (`

**Comment by camspiers** [comment]
Seems like a reasonable addition.

**Comment by frederikbosch** [comment]
@camspiers Thanks, just removed the space.

**Comment by sagikazarmark** [comment]
:+1: I think you should open a PR against nextrelease as well.

**Comment by frederikbosch** [comment]
@sagikazarmark Done.

**Comment by sagikazarmark** [comment]
:+1:

**Pull request merged**